lib/logger.c

```diff
@@ -122,9 +125,9 @@ void log_log(int level, const char *file, int line, const char *fmt, ...) {
     /* Log to file */
     if (L.fp) {
         va_list args;
-        char buf[32];
-        buf[strftime(buf, sizeof(buf), "%Y-%m-%d %H:%M:%S", lt)] = '\0';
-        fprintf(L.fp, "%s %-5s %s:%d: ", buf, level_names[level], file, line);
+        fprintf(
+            L.fp, "%s %-5s %-5s %-16s %4d: ",
```

Comment:
This is the log format. Maybe this should be a macro so that all can share the format.